test/cli/cli.test.js

```diff
@@ -29,6 +29,22 @@ describe('CLI', () => {
       .catch(done);
   });
 
+  it('--quiet', async (done) => {
+    const output = await testBin('--quiet');
+    expect(output.code).toEqual(0);
+    expect(output.stdout.split('\n').length === 3).toBe(true);
+    expect(
+      output.stdout.includes('Project is running at http://localhost:8080/')
```

Comment:
Please use `ports-map` to get a port that is not 8080, then specify it with `--port`. It looks like there are some unused `cli` ports here that can be used: https://github.com/webpack/webpack-dev-server/blob/master/test/ports-map.js#L6

Reply:
Gotcha, Will look into it soon.